applications/asset_tracker/configuration/mcuboot_prj.conf

```diff
@@ -0,0 +1,43 @@
+### This is a copy of what can be found in boot/zephyr/prj.conf in MCUBoot
```

Comment:
Why is a complete copy of this file needed ?
Isn't it possible to provide an overlay file with just:
```
### Added to ensure compatibility with older Thingy:91 firmware
CONFIG_BOOT_SIGNATURE_TYPE_RSA=y
CONFIG_BOOT_SIGNATURE_KEY_FILE="root-rsa-2048.pem"
```

or set those two values directly with:
```
set(mcuboot_CONFIG_BOOT_SIGNATURE_TYPE_RSA "y")
set(mcuboot_CONFIG_BOOT_SIGNATURE_KEY_FILE "root-rsa-2048.pem")
```

Reply:
I tried with an overlay file. But I'll try to set them directly in cmake

Reply:
Thanks for the help it's much better now.